include/ginkgo/config.hpp.in

```diff
@@ -82,6 +82,18 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 // clang-format on
 
 
+/* Is TAU available for Profiling? */
+// clang-format off
```

Comment:
Are these clang-format actually required. Removing them doesn't seem to change anything on my system.

Reply:
yes, removing the comments leads to a space before the trailing @ sign